src/scripts

```diff
@@ -59,4 +59,248 @@ p({
       print(t.getMessage());
     }
   """
-})
\ No newline at end of file
+})
+
+p({
```

Comment:
Perhaps move these test scripts into 
src/foam/i18n/scripts.jrl 

Also, where are the scripts to extract all the messages and generate the xliff file? 
Assume it's similar to these without the locale, variant, and target.